scripts/usage_reporting/README.md

```diff
@@ -25,6 +25,8 @@ Example:
 ```
 go run event.go process_events.go \
   -org "<ORG_UUID>" \
+  -start "2017-10-01T00:00:00Z" \
+  -finish "2017-10-31T23:59:59Z" \
```

Comment:
Should we use 1st of the following month for simplicity? It's easier to read and reduces the chance that we'd make an off-by-one error with the number of days in the current month. I don't think it will have any effect on the results.

Reply:
I think that would be fine.